doc/manual/manual.bib

```diff
@@ -423,15 +423,13 @@ @Article{BSTOL02
                   Oncken, O. and Lavier, L. L.},
   title =        {Numerical models of crustal scale convection and partial
                   melting beneath the {A}ltiplano-{P}una plateau},
-  journal =      {??},
-  year =         {??},
-  OPTkey =       {},
-  OPTvolume =    {??},
-  OPTnumber =    {},
-  OPTpages =     {??},
-  OPTmonth =     {},
-  OPTnote =      {},
-  OPTannote =    {}
+  year = {2002},
+  month = {06},
+  pages = {373-388},
+  volume = {199},
+  journal = {Earth and Planetary Science Letters},
+  doi = {10.1016/S0012-821X(02)00597-6}
+                  }
```

Comment:
is this } correct here?

Reply:
Nice catch. Interesting that my parser did not catch that!